### PR TITLE
Version bump (1.0.0)

### PR DESCRIPTION
**Pull request by CastagnaIT**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

After the latest major developments that i have achieved, concerning:

- implementation of msl profile switch
- implementation of watched status synchronization with netflix in a two way mode

we can declare the first version

will not be without problems, the development of this part is quite difficult, especially due to lack of documentation, there are too many factors to take into account.
The bug fixes take time and a lot of testing by different devices and also different countries, i can't do always by myself then when problems are found will be solved

### CODE REVIEW

**Comment by dagwieers** [comment]
I would keep it simple and declare it **v1.0.0** ;-)

**Comment by dagwieers** [comment]
Can you please rebase this PR?

**Comment by CastagnaIT** [comment]
it's not a problem to follow that kind of versioning

i do not rebase this branch beacause is needed to merge https://github.com/CastagnaIT/plugin.video.netflix/pull/486

I still have strange problems with ctx menus
"resume from"
"start from beginning"
when i keep enabled watched status sync, which is wasting my time
in my computer were working, then last night the HTPC wasn't working
i have to try updating kodi on HTPC, i don't know😕

**Comment by dagwieers** [comment]
> i do not rebase this branch beacause is needed to merge #486

If you rebase this branch, at least it would include all commits since 10 days.

> I still have strange problems with ctx menus

Hmm, I would expect issues with "reset position" and "mark as (un)watched", not with "resume from" or "start from beginning"?

Unfortunately this is because Kodi does not gives us the proper tools to make it work well. There are 2 options here, both requiring changes in Kodi:
- An event handler that is called when "reset position" or "mark as (un)watched" is called
- Timestamps for "resumetime" and "playcount", both in Kodi and at Netflix so asynchronous synchronisation is possible

The former is prefered over the latter though. Disabling the context menu items is another option.

In our add-on we have this marked as a known issue in Kodi that we currently cannot fix.

**Comment by CastagnaIT** [comment]
yes with "reset position" and "mark as (un)watched" also i have problems,
but i added a new custom menu in order to change the status.

I tried to propose hiding the non-working menus, only when the "override" is used
https://github.com/xbmc/xbmc/issues/17372
i have also attach a commit on how it could be implemented

but it wasn't accepted because they want to keep those menus even if they don't work, because of "coherence" between addons menus?!...

the only alternative is to review the entire system and also implement a python callback,
i see it as a complex thing and i don't think it will be fixed for Kodi 19, probably be broken for all the eternity

at least if these menus were hidden for the time being, we could have one thing cleaner and less confusion for the users, and in the future it could have been fixed with tranquility

**Comment by dagwieers** [comment]
@CastagnaIT Thanks for the heads up. I agree, let's get more stakeholders involved to make it clear we need a solution.

IMO video-on-demand add-on developers should unite around missing functionality in Kodi, because Kodi is no longer adapted to today's streaming needs. https://github.com/pietje666/plugin.video.vrt.nu/wiki/Kodi

**Comment by CastagnaIT** [comment]
sidenote: for the problem of "resume from" I've done over a dozen tests and happens randomly,
i also tried to exclude to set the "setInfo" in the listitem to be played and also exclude all background netflix operations, but they don't affect. I really don't know what the problem is...

I analyzed again the entire source code,
and i see nothing unusual here, I'm starting to think it's a Kodi bug

**Comment by CastagnaIT** [comment]
i have also rebased this branch

File: addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.netflix" name="Netflix" version="0.16.4" provider-name="libdev + jojo + asciidisco + caphm + castagnait">
+<addon id="plugin.video.netflix" name="Netflix" version="1.00.0" provider-name="libdev + jojo + asciidisco + caphm + castagnait">
```
Comment (tweimer) [inline addon.xml]: Why not "1.0.0" instead of "1.00.0"?

Comment (CastagnaIT) [reply addon.xml]: habits 😅 yes i can change

**Pull request merged**